docs/guide/counter/cmd/countercli/commands/counter.go

```diff
@@ -55,6 +58,10 @@ func counterTx(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	//get the nonce accounts
+	var addr []byte
```

Comment:
We need an easy way of getting the address from the name flag here! Needs to be assigned to this value... how on earth in the new system? This was some of the functionality that I programmed in 0.6 ... ahh